tlatools/org.lamport.tlatools/test/tlc2/value/StringDeserializeTLCTest.java

```diff
@@ -0,0 +1,21 @@
+package tlc2.value;
```

Comment:
Please add a copyright header.

Reply:
I attempted this. I reproduced the MIT license part from another test file I've touched, but removed the Microsoft part (I am not affiliated right now) and added my name as contributor, like it seems to be the procedure.